### PR TITLE
[cosmwasm]  Use ParsedVaa from wormhole contract

### PR DESCRIPTION
**Pull request by guibescos**
I noticed this while playing around with the cosmos contract. 
We were copy pasting ParsedVAA and WormholeQueryMsg instead of importing them from cw_wormhole.
The reason for this seems to be incompatiblity with osmosis. However I was able to build properly with osmosis, so it think the reason doesn't exist anymore and we can import from the wormhole crate.

### CODE REVIEW

File: target_chains/cosmwasm/contracts/pyth/src/contract.rs [hunk 1]
```diff
@@ -1095,28 +1095,28 @@ mod test {
...
     fn check_sufficient_fee(deps: &Deps, data: &[Binary]) {
         let mut info = mock_info("123", coins(100, "foo").as_slice());
-        let result = is_fee_sufficient(&deps, info.clone(), &data);
+        let result = is_fee_sufficient(deps, info.clone(), data);
         assert_eq!(result, Ok(true));
 
         // insufficient fee in base denom -> false
         info.funds = coins(50, "foo");
-        let result = is_fee_sufficient(&deps, info.clone(), &data);
+        let result = is_fee_sufficient(deps, info.clone(), data);
```
Comment (guibescos) [inline target_chains/cosmwasm/contracts/pyth/src/contract.rs:1103]: Clippy did this

File: target_chains/cosmwasm/contracts/pyth/src/wormhole.rs [hunk 2]
```diff
@@ -1,46 +0,0 @@
-// These types are copied from the Wormhole contract. See the links with each type to see the original code
-// The reason to do so was dependency conflict. Wormhole contracts were using a very old version of a dependency
-// which is not compatible with the one used by osmosis-sdk. And since we weren't using anything else from
```
Comment (guibescos) [inline target_chains/cosmwasm/contracts/pyth/src/wormhole.rs:3]: Seems like this comment is outdated

**Review by jayantk — APPROVED** [review]
I think if it builds, it should be fine?

**Comment by m30m** [comment]
Thanks! It would be great if you can run `Release Pyth Cosmwasm Contract` workflow against this branch just to make sure everything works.

File: target_chains/cosmwasm/contracts/pyth/Cargo.toml [hunk 3]
```diff
@@ -40,6 +40,7 @@ cosmwasm-schema = "1.1.9"
 osmosis-std = "0.15.2"
 pyth-sdk-cw = { path = "../../sdk/rust" }
 pythnet-sdk = { path = "../../../../pythnet/pythnet_sdk" }
+wormhole-cosmwasm =  {git = "https://github.com/wormhole-foundation/wormhole"}
```
Comment (ali-bahjati) [inline target_chains/cosmwasm/contracts/pyth/Cargo.toml:43]: can you please lock it to a commit?
I think wormhole folks should really consider releasing their packages.

**Pull request merged**